### PR TITLE
Hocs-6145: Helm deployment

### PR DESCRIPTION
**Pull request by rjweeks**

### CODE REVIEW

File: helm/hocs-cms-data-migrator/templates/_envs.tpl [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  valueFrom:
+    secretKeyRef:
+      name: {{.KUBE_NAMESPACE}}-case-migrator-sqs
+      key: secret_access_key
+- name: AWS_SQS_CASE_CREATOR_URL
+  valueFrom:
+    secretKeyRef:
+      name: {{.KUBE_NAMESPACE}}-case-migrator-sqs
+      key: sqs_url
+- name: CMS_EXTRACT_COMPLAINTS=enabled
```
Comment (SWhitlam) [inline helm/hocs-cms-data-migrator/templates/_envs.tpl:71]: This doesn't look valid, also could be a boolean?
```suggestion
- name: CMS_EXTRACT_COMPLAINTS_ENABLED
  value: true
```

File: helm/helm-deploy.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+helm upgrade ${CHART_NAME} \
+ ./helm/${CHART_NAME} \
+--install \
+--reset-values \
+--timeout 10m \
+--history-max 3 \
+--namespace ${KUBE_NAMESPACE} \
+--set version=${VERSION} $*
```
Comment (SWhitlam) [inline helm/helm-deploy.sh:10]: You won't need the $* here if you plan to run this manually
```suggestion
--set version=${VERSION}
```

File: helm/hocs-cms-data-migrator/templates/_envs.tpl [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+- name: CMS_DB_PASSWORD
+  valueFrom:
+    secretKeyRef:
+      name: {{.KUBE_NAMESPACE}}-mssql
+      key: password
+- name: POSTGRESG_DB_HOST
+  valueFrom:
+    secretKeyRef:
+      name: {{.KUBE_NAMESPACE}}-postgres
+      key: password
```
Comment (SWhitlam) [inline helm/hocs-cms-data-migrator/templates/_envs.tpl:35]:
```suggestion
      key: host
```

File: helm/hocs-cms-data-migrator/templates/_envs.tpl [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+- name: POSTGRES_DB_PORT
+  valueFrom:
+    secretKeyRef:
+      name: {{.KUBE_NAMESPACE}}-postgres
+      key: port
+- name: POSTGRES_DB_NAME
+  valueFrom:
+    secretKeyRef:
+      name: {{.KUBE_NAMESPACE}}-postgres
+      key: default_db
```
Comment (SWhitlam) [inline helm/hocs-cms-data-migrator/templates/_envs.tpl:45]:
```suggestion
      key: name
```

File: helm/hocs-cms-data-migrator/templates/_envs.tpl [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+- name: POSTGRES_DB_USERNAME
+  valueFrom:
+    secretKeyRef:
+      name: {{.KUBE_NAMESPACE}}-postgres
+      key: username
```
Comment (SWhitlam) [inline helm/hocs-cms-data-migrator/templates/_envs.tpl:50]:
```suggestion
      key: user_name
```

Comment (rjweeks) [reply helm/hocs-cms-data-migrator/templates/_envs.tpl]: Updated.

File: helm/hocs-cms-data-migrator/templates/_envs.tpl (hunk 4 above)
Comment (rjweeks) [reply helm/hocs-cms-data-migrator/templates/_envs.tpl]: Updated.

File: helm/hocs-cms-data-migrator/templates/_envs.tpl (hunk 3 above)
Comment (rjweeks) [reply helm/hocs-cms-data-migrator/templates/_envs.tpl]: Updated.

File: helm/hocs-cms-data-migrator/templates/_envs.tpl (hunk 1 above)
Comment (rjweeks) [reply helm/hocs-cms-data-migrator/templates/_envs.tpl]: Updated.

**Review by SWhitlam — APPROVED** [review]

**Pull request merged**